Makefile

```diff
@@ -85,3 +85,7 @@ model/marshal_fastjson.go: model/model.go
 
 scripts/Dockerfile-testing: $(wildcard module/*)
 	go generate ./scripts
+
+.PHONY: check-vanity-import
+check-vanity-import:
```

Comment:
can you please move this up with the other conditional targets at line 15? and define an empty target for Go < 1.12